docs/plugins/discovery-gce.asciidoc

```diff
@@ -56,8 +66,13 @@ The following gce settings (prefixed with `cloud.gce`) are supported:
 
  `max_wait`::
 
-     The maximum elapsed time in milliseconds after the client instantiating retry. If the time elapsed goes past the
-     `max_wait`, client stops to retry. Defaults to 15 minutes (900000 milliseconds).
+     The maximum elapsed time after the client instantiating retry. If the time elapsed goes past the
+     `max_wait`, client stops to retry. A negative value means that it will wait indefinitely. Defaults to `-1s`.
```

Comment:
I am not sure whether the first sentence is needed at all. Also, I think the name `max_wait` is a bit misleading when retry is involved.